### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2143]
Fix all accessibility issues:

- Accordion
- Alert
- Button
- Card
- Chip
- Content Tree
- Data Table
- Dropdown
- Pagination
  - #2043 
- Slider/Range
- Toast

**Comment by coliff** [comment]
FIXED:

- Toast
  - All toasts have `role="status"`, but some toasts may be alerts. The role should be configurable.
